microdnf/main.cpp

```diff
@@ -71,15 +72,13 @@ static bool parse_args(Context & ctx, int argc, char * argv[]) {
     setopt->set_description(R"**(Override a configuration option from the configuration file. To override configuration options for repositories, use repoid.option for  the
               <option>.  Values  for configuration options like excludepkgs, includepkgs, installonlypkgs and tsflags are appended to the original value,
               they do not override it. However, specifying an empty value (e.g. --setopt=tsflags=) will clear the option.)**");
-
-    // --setopt option support
     setopt->set_parse_hook_func([&ctx](
                                [[maybe_unused]] ArgumentParser::NamedArg * arg,
                                [[maybe_unused]] const char * option,
                                const char * value) {
         auto val = strchr(value+1, '=');
         if (!val) {
-            throw std::runtime_error(std::string("setopt: Badly formated argument value") + value);
+            throw std::runtime_error(fmt::format("setopt: Badly formated argument value \"{}\"", value));
```

Comment:
What about translation of the error?

Reply:
We should also set a standard to highlight the value in message: `'{}'`, `\"{}\"`, ...

Reply:
@j-mracek 
- Yes translation. Good point. But this is microdnf not libdnf. I need to prepare translation support in microdnf. Probably another commit.
- We chose double quotes at the meeting.